src/sc/proofverifier.cpp

```diff
@@ -46,129 +44,209 @@ void CScProofVerifier::LoadDataForCertVerification(const CCoinsViewCache& view,
         bt.amount = btout.nValue;
     }
 
-    certData.quality = scCert.quality; //Currently quality not yet accounted for in proof verifier
-    if (sidechain.fixedParams.constant.is_initialized())
-        certData.constant = sidechain.fixedParams.constant.get();
-    else
-        certData.constant = CFieldElement{};
+    for (auto entry: scCert.vFieldElementCertificateField)
+    {
+        CFieldElement fe{entry.getVRawData()};
+        certData.vCustomFields.push_back(fe);
+    }
+    for (auto entry: scCert.vBitVectorCertificateField)
+    {
+        CFieldElement fe{entry.getVRawData()};
```

Comment:
Same here instead of passing `entry.getVRawData()` we should use `GetFieldElement` method that will return the merkle root of the compressed bitvector.

Reply:
Same as above